src/vs/workbench/api/browser/mainThreadEditor.ts

```diff
@@ -62,25 +62,30 @@ export class MainThreadTextEditorProperties {
 
 		let cursorStyle: TextEditorCursorStyle;
 		let lineNumbers: RenderLineNumbersType;
+		let rulers: (number | IRulerOption)[];
```

Comment:
This type is used pretty frequently, I wasn't sure if I should make it a type in `common/config/editorOptions` and import it at the different locations or not.